apps/settings/js/startup.js

```diff
@@ -276,16 +276,7 @@
      */
     _getInitialPanelId: function as_getInitialPanelId() {
       return new Promise(function(resolve) {
-        if (navigator.mozHasPendingMessage('activity')) {
-          // Load activity handler only when we need to handle it.
-          LazyLoader.load(['js/activity_handler.js'], function ah_loaded() {
-            window.ActivityHandler.ready().then(function ah_ready() {
-              resolve(window.ActivityHandler.targetPanelId);
-            });
-          });
-        } else {
-          resolve('root');
-        }
```

Comment:
We need to file a follow-up bug to replace this functionality. Maybe with a Web Activities alternative or possibly deep links.